centreon/src/Core/Security/AccessGroup/Application/Repository/WriteAccessGroupRepositoryInterface.php

```diff
@@ -58,4 +58,24 @@ public function addLinksBetweenHostGroupAndAccessGroups(int $hostGroupId, array
      * @param AccessGroup[] $accessGroups
      */
     public function addLinksBetweenServiceGroupAndAccessGroups(int $serviceGroupId, array $accessGroups): void;
+
+    /**
+     * Add links between a "Host Group" and multiple "Resource Ids".
+     *
+     * @param int $hostGroupId
+     * @param int[] $resourceIds
+     */
+    public function addLinksBetweenHostGroupAndResourceIds(int $hostGroupId, array $resourceIds): void;
+
+    /**
+     * Sets the changed flag to 1
+     *
+     * @param AccessGroup[] $accessGroupIds
+     */
+    public function updateAclGroupsFlag(array $accessGroupIds): void;
+
+    /**
+     * Update the changed flag to 1 of the ACL resources.
+     */
+    public function updateAclResourcesFlag(): void;
```

Comment:
the value set to 1 is proper to implementation with MariaDB and MySQL, Interface should be agnostic of DBMS,

Maybe remove the the `1` from the phpdoc and/or rename the method to explictly indicate that the flag has been updated for a "ready to reload" state